actions-omitted.yaml

```diff
@@ -1,7 +1,16 @@
 # Avoid syncing certain actions to certain repos. (For example, go testing from non-go repos.)
 # This uses a glob prefix match (adds * to the end of each omit), so no need to specify '.yaml'
 'knative/website':
-    omit:
-    - 'knative-go-'
-    - 'knative-releasability'
-    - 'knative-release-notes'
\ No newline at end of file
+  omit:
+  - 'knative-go-'
+  - 'knative-releasability'
+  - 'knative-release-notes'
+'knative-sandbox/kperf'
```

Comment:
The formatting looks a bit messed up to me.

Reply:
good catch :)